### DISCUSSION TITLE
writing/2024/06/01/advice-to-young-people/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#62] [General]
# writing/2024/06/01/advice-to-young-people/

Notes about my hobbies and other things I find interesting.

https://jxnl.co/writing/2024/06/01/advice-to-young-people/

**Comment by lilium333** [comment]
"I remember that video where they were asking this Olympic weightlifter what matters more, talent or hard work. And he said something along the lines of, "At the national level, everyone is talented, but when you get to the Olympics hard work is the only thing that matters." There are many injuries and challenges, and the people who get it are the people who want it the most." 

Do you still have this video?

**Comment by sandipan1** [comment]
a lot of this article feels like someone just put words to my thoughts. I resonate a lot reading this..very grateful !!

**Comment by eperez28** [comment]
Thank you Jason for taking the time to distill this piece. I found the section on self-worth delicious, as it’s something I’ve been working hard on earlier this year. Your note on dancing in public is liberating too!

**Comment by Dhanushkmr** [comment]
I needed this. thank you.
